app/presenters/keyword_presenter.rb

```diff
@@ -5,14 +5,46 @@ def initialize(keyword)
     @keyword = keyword
   end
 
+  delegate :id, to: :keyword, prefix: true
```

Comment:
How about grouping the `delegate`'s together?

Reply:
Fixed in [`5f004ca` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/5f004cae025caf7b86ce910239bd5478850fb86a)